app/src/renderer/components/staking/TabParameters.vue

```diff
@@ -102,19 +102,19 @@ export default {
       inflation_rate_change: `Maximum annual change in inflation rate`,
       inflation_max: `Maximum inflation rate`,
       inflation_min: `Minimum inflation rate`,
-      goal_bonded: `Goal for percentage of bonded atoms`,
-      unbonding_time: `Time to complete an unbonding delegation and claim rewards`,
+      goal_bonded: `Goal for percentage of delegated atoms`,
+      unbonding_time: `Time to complete an undelegation transaction and claim rewards`,
```

Comment:
```suggestion
      unbonding_time: `Time to complete an unbonding delegation transaction and claim rewards`,
```

Reply:
we're not using the B words anymore

Reply:
ohh really ? I didn't know

Reply:
t'sall good. https://github.com/cosmos/voyager/issues/1454#issue-368813264